docs/source/user-guide/tuning.md

```diff
@@ -141,30 +191,22 @@ It must be set before the Spark context is created. You can enable or disable Co
 at runtime by setting `spark.comet.exec.shuffle.enabled` to `true` or `false`.
 Once it is disabled, Comet will fall back to the default Spark shuffle manager.
 
-### Shuffle Mode
+### Shuffle Implementations
 
-Comet provides three shuffle modes: Columnar Shuffle, Native Shuffle and Auto Mode.
+Comet provides two shuffle implementations: Native Shuffle and Columnar Shuffle. Comet will first try to use Native
+Shuffle and if that is not possible it will try to use Columnar Shuffle. If neither can be applied, it will fall
+back to Spark for shuffle operations.
```

Comment:
It would be helpful to say this is the default auto mode. The new explanation does not have `auto` keyword.

Reply:
Shuffle mode is an undocumented internal config that the user should never change, so it seemed confusing to discuss in the user guide?